scipy/ndimage/src/ni_morphology.c

```diff
@@ -703,14 +703,14 @@ int NI_DistanceTransformBruteForce(PyArrayObject* input, int metric,
                     temp = temp->next;
                 }
                 if (distances)
-                    *(UInt32*)pd = distance;
+                    *(npy_int32*)pd = distance;
```

Comment:
This looks correct, but just checking: you changed ``uint32`` to ``int32`` on purpose here right?